src/pages/Clusters.js

```diff
@@ -342,7 +342,7 @@ const Environments = () => {
 
 export const navPaths = [
   {
-    name: 'clusters',
+    name: 'environments',
     path: '/clusters',
```

Comment:
Mind adding a comment here explaining why the path is different than the name now? (Just for future breadcrumbs)